diff --git a/Makefile b/Makefile
index 439f7ff..5a5a499 100644
--- a/Makefile
+++ b/Makefile
@@ -72,7 +72,7 @@ bootstrap:
 	recess --compile ${BOOTSTRAP_RESPONSIVE_LESS} > bootstrap/css/bootstrap-responsive.css
 	recess --compress ${BOOTSTRAP_RESPONSIVE_LESS} > bootstrap/css/bootstrap-responsive.min.css
 	cat js/bootstrap-transition.js js/bootstrap-alert.js js/bootstrap-button.js js/bootstrap-carousel.js js/bootstrap-collapse.js js/bootstrap-dropdown.js js/bootstrap-modal.js js/bootstrap-tooltip.js js/bootstrap-popover.js js/bootstrap-scrollspy.js js/bootstrap-tab.js js/bootstrap-typeahead.js js/bootstrap-affix.js > bootstrap/js/bootstrap.js
-	uglifyjs -nc bootstrap/js/bootstrap.js > bootstrap/js/bootstrap.min.tmp.js
+	uglifyjs bootstrap/js/bootstrap.js -mc > bootstrap/js/bootstrap.min.tmp.js
 	echo "/*!\n* Bootstrap.js by @fat & @mdo\n* Copyright 2012 Twitter, Inc.\n* http://www.apache.org/licenses/LICENSE-2.0.txt\n*/" > bootstrap/js/copyright.js
 	cat bootstrap/js/copyright.js bootstrap/js/bootstrap.min.tmp.js > bootstrap/js/bootstrap.min.js
 	rm bootstrap/js/copyright.js bootstrap/js/bootstrap.min.tmp.js
@@ -98,4 +98,4 @@ watch:
 	watchr -e "watch('less/.*\.less') { system 'make' }"
 
 
-.PHONY: docs watch gh-pages
\ No newline at end of file
+.PHONY: docs watch gh-pages bootstrap
diff --git a/less/bootstrap.less b/less/bootstrap.less
index 14bb3f0..83cfd7a 100644
--- a/less/bootstrap.less
+++ b/less/bootstrap.less
@@ -15,6 +15,9 @@
 @import "variables.less"; // Modify this for custom colors, font-sizes, etc
 @import "mixins.less";
 
+// Custom Addons
+@import "../../../allplayers.less";
+
 // Grid system and page structure
 @import "scaffolding.less";
 @import "grid.less";
@@ -61,3 +64,6 @@
 
 // Utility classes
 @import "utilities.less"; // Has to be last to override when necessary
+
+// Custom Addons (for CSS overrides)
+@import "../../../allplayers.less";
diff --git a/less/responsive.less b/less/responsive.less
index aa28baa..159b426 100644
--- a/less/responsive.less
+++ b/less/responsive.less
@@ -21,6 +21,9 @@
 @import "variables.less"; // Modify this for custom colors, font-sizes, etc
 @import "mixins.less";
 
+// Custom Addons
+@import "../../../allplayers.less";
+
 
 // RESPONSIVE CLASSES
 // ------------------
